javascript/dynamicpage.js

```diff
@@ -0,0 +1,46 @@
+$(function() {
```

Comment:
You use `$(function() {});` here and `$().ready(function() {});` later in signup_form.js. They are equivalent syntaxes but the inconsistency is confusing. `$(document).ready(function() {});` is also equivalent and a little more explicit as to what you are doing. I think we should use that.